<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signal_generator_connect</class>
 <widget class="QWidget" name="signal_generator_connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal Generator Connection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="connection_layout">
     <item>
      <layout class="QHBoxLayout" name="devices_layout" stretch="1,2,1">
       <item>
        <widget class="QLabel" name="devices_label">
         <property name="text">
          <string>Devices</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="devices_combobox"/>
       </item>
       <item>
        <widget class="QPushButton" name="refresh_button">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="connect_layout">
       <item>
        <widget class="QPushButton" name="connect_button">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect_button">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="status_label">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
